gatsby-browser.js

```diff
@@ -49,6 +49,14 @@ const GlobalStyles = createGlobalStyle`
     p {
       margin-bottom: 1.5rem;
     }
+
+    a {
+      font-family: ${prop => prop.theme.fonts.links};
+    }
+
+    .category {
+      font-family: ${prop => prop.theme.fonts.links};
+    }
```

Comment:
You can define both in the same time:
```suggestion
    .category, a {
      font-family: ${prop => prop.theme.fonts.links};
    }
```